### DISCUSSION TITLE
My account is not public and it returns 404 when visiting my profile URL

### DISCUSSION THREAD

**Discussion by tarekrahman3** [community/community#61004] [Other Feature Feedback, Questions, & Ideas]
Suddenly GitHub made my profile private? I did not receive any kind of notification or email regarding this. I can't refer my potential clients. This happened maybe within last three or four days. What do I do?

**Comment by ilnitkosta** [comment]
Bro, maybe it was just kind of error, because your profile is absolutely okay right now.
